### PR TITLE
Update readme with correct reference links

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f890377](https://github.com/OfficeDev/office-scripts-docs/commits/f8903771d31d47ae746334272ce99a1b570349da)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[README.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJerabek-patch-1/README.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912131558271449-54%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2F9j84DBU8908eP9%2B1ptNOdp4OHSc%2FpY2Vf5BJuvWUUU%3D&st=2019-12-13T15%3A54%3A32Z&se=2020-01-13T15%3A59%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to the Office Scripts documentation repo. In this repository, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you view this content on [docs.microsoft.com](https://docs.microsoft.com/office-scripts-docs).
 
-> **Note**: You can find the reference documentation source files for Office JavaScript API (on which Office Scripts run) in the [office-js-docs-reference](https://github.com/OfficeDev/office-js-docs-reference) repository. These APIs contain all the potential Excel operations for your scripts.
+> **Note**: You can find the reference documentation source files for the Office Scripts API (on which Office Scripts run) in the [office-scripts-docs-reference](https://github.com/OfficeDev/office-scripts-docs-reference) repository. These APIs contain all the potential Excel operations for your scripts.
```
Comment (ElizabethSamuel-MSFT) [inline README.md]:
```suggestion
> **Note**: You can find the reference documentation source files for the Office Scripts API in the [office-scripts-docs-reference](https://github.com/OfficeDev/office-scripts-docs-reference) repository. These APIs contain all the potential Excel operations for your scripts.
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ We also encourage you to fork, make the fix, and do a pull request of your propo
 
 If your issue is not related to the Office Scripts documentation, please post it to one of the following channels instead:
 
-- To ask a question about designing Office Scripts or the Office.js API that runs Office Scripts, post your question to Stack Overflow and tag it with the "office-scripts" tag (https://stackoverflow.microsoft.com/questions/tagged/office-scripts).
+- To ask a question about designing Office Scripts or the Office.js API that runs Office Scripts, post your question to Stack Overflow and tag it with the "office-scripts" tag (https://stackoverflow.com/questions/tagged/office-scripts).
 - To report an issue with the Office.js API, create the issue in the OfficeDev/office-js repository (https://github.com/OfficeDev/office-js), which members of the product team monitor for customer-reported issues.
```
Comment (ElizabethSamuel-MSFT) [inline README.md]:
```suggestion
- To report an issue with the Office.js API, create the issue in the [OfficeDev/office-js repository](https://github.com/OfficeDev/office-js), which members of the product team monitor for customer-reported issues.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4849a5e](https://github.com/OfficeDev/office-scripts-docs/commits/4849a5eeea91cbe7899da78510028e40e20cc18b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[README.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJerabek-patch-1/README.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912131814106256-54%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8ckEsnLXCN%2FRZ7FNIR81Mk3hCdPeMAA1YGApiaIQ4Vo%3D&st=2019-12-13T18%3A10%3A14Z&se=2020-01-13T18%3A15%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f1d4112](https://github.com/OfficeDev/office-scripts-docs/commits/f1d4112db3dc8241564ecdbf51fe1bf8a7d0ca6c)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-45a7770c)

<a id="45a7770c"></a>
###
 - **[Error]** 
`````
Cannot sync git repo to specified commit because branch AlexJerabek-patch-1 has been deleted or has been force pushed: fatal: couldn't find remote ref refs/heads/AlexJerabek-patch-1

`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C13%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C201912131814342105-54%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=riXncfdgpX3ZzjTX%2FvgyeFsq21Qnj8r2lm7WCc5ESLk%3D&st=2019-12-13T18%3A10%3A25Z&se=2020-01-13T18%3A15%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**